upgrade master builds/unstable

boferri opened the pull request:
- incl. SAPI, UUID, Hibernate + JPA + Java upgrade, PNX + Marc21 schema

Commit:
[DD-885] added DB reset to maintain DB service

fniederl committed:
Merge pull request #13 from zazi/sprint-28/dd-885

[DD-885] added DB reset to maintain DB service

Commit:
[DD-886] increased logging level

Commit:
[DD-886] separated affected tests (+ decreased logging level)

Commit:
[DD-886] fixed deprecated API usage

knutwalker committed:
Merge pull request #14 from zazi/sprint-28/dd-886

[DD-886] fixed test failures on jenkins

Commit:
[DD-886] RIP POJOCLASSIDTYPE

Commit:
[WIP][DD-886] towards UUID

Commit:
[DD-886] resolved imports

Commit:
[WIP][DD-886] towards UUID (started removing ID)

knutwalker committed:
Merge pull request #15 from zazi/sprint-28/dd-514

[DD-886] RIP POJOCLASSIDTYPE

Commit:
[WIP][DD-886] towards UUID (ID -> UUID + adapted schema.sql)

Commit:
[WIP][DD-886] made default constructor (without parameters) package protected (+ adapted usage/ entity creation)

Commit:
[WIP][DD-886] made default constructor (without parameters) package protected (+ adapted usage/ entity creation) pt. 2 (tbc)

Commit:
[WIP][DD-886] adapted usage/ entity creation (made POJO unit tests mainly working) pt. 3 (tbc)

Commit:
[WIP][DD-886] adapted usage/ entity creation (made persistence module tests mainly working) pt. 4 (tbc)

Commit:
[WIP][DD-886] adapted usage/ entity creation (made converter module compilable) pt. 5 (tbc)

Commit:
[WIP][DD-886] adapted metadata repository data init scripts

Commit:
[WIP][DD-886] moved static uuids of schemata and data models to more appropriated places

Commit:
[WIP][DD-889] implemented id to uuid converter for test json resources (incl. dummy test for converter module)

Commit:
[DD-889] dummy test for id to uuid test json resources transformation

Commit:
[WIP][DD-514] fixed XMLSourceResourceGDMStmtsFlowTests

knutwalker committed:
Fix Deserializer that would throw away accumulated values

Commit:
Merge pull request #1 from knutwalker/sprint-28/dd-514-new

Fix Deserializer that would throw away accumulated values

knutwalker committed:
Fix CSVTransformationTest.testCSVEndToEndWithEmptyValues

Commit:
Merge pull request #2 from knutwalker/sprint-28/dd-514-new

Fix CSVTransformationTest.testCSVEndToEndWithEmptyValues

knutwalker committed:
Fix XMLTransformationFlowTest.testMabxmlFilterWithRegexTask

knutwalker committed:
Fix MorphScriptBuilderTest

data was complete, only ordering was off

Commit:
Merge pull request #3 from knutwalker/sprint-28/dd-514-new

Fix XMLTransformationFlowTest.testMabxmlFilterWithRegexTask

knutwalker committed:
[DD-889] Fix CSVTransformationFlowTest.testCSVMultipleMappingsWithAlmostAllFunctions

Commit:
[WIP][DD-514] fixed controller module compilation erros + fixed some test failures

Commit:
Merge branch 'sprint-28/dd-514-new' of github.com:zazi/dswarm into sprint-28/dd-514-new

Commit:
Merge pull request #4 from knutwalker/sprint-28/dd-514-new

[DD-889] Fix CSVTransformationFlowTest.testCSVMultipleMappingsWithAlmost...

knutwalker committed:
[DD-889] Use BLOBs for JSON string fields

knutwalker committed:
[DD-889] Remove usage of File in tests where it is not really necessary

knutwalker committed:
[DD-889] Rework TransformationDeserializer

knutwalker committed:
[DD-889] Better usage of String.format

Commit:
Merge pull request #5 from knutwalker/sprint-28/dd-514-new

FInal test failures fixed for DD-889

Commit:
[DD-893]PNX schema based on subschema branch, throws nullpointer however

knutwalker committed:
[DD-898] Various Version updates and version property refactoring

Commit:
[WIP][DD-893] enhanced XmlSchemaParser for enabling PNX schema parsing

Commit:
Merge branch 'sprint-28/dd-514-new' into sprint-29/DD-893

Conflicts:
	converter/src/test/java/org/dswarm/converter/schema/test/BuildInitInternalSchemaScriptTest.java

Commit:
[WIP][DD-893] towards PNX schema into internal data sql script

knutwalker committed:
[DD-898] Add some linting compiler flags

knutwalker committed:
[DD-898] Remove dependency from BasicJPAServiceTestUtils on BasicJPAServiceTest

knutwalker committed:
[DD-898] Reformat/-structure Hibernate configuration

Commit:
[WIP][DD-893] change SCHEMA_ATTRIBUTE_PATH_INSTANCES column in DATA_SCHEMA to LONGBLOB

Commit:
[DD-893] updated init_internal_schema.sql script with PNX schema

knutwalker committed:
[DD-898] One equals to rule them all

Commit:
[DD-903] eliminated *ResourceUtils

knutwalker committed:
[DD-898] Consisten UUID for internal schema and test data

Commit:
Merge pull request #17 from knutwalker/sprint-29/dd-898

[DD-898] Towards Hibernate update

knutwalker committed:
Merge pull request #18 from zazi/sprint-29/dd-903

[DD-903] eliminated *ResourceUtils in controller module

Commit:
[DD-898] some minor fixes/enhancements re. hibernate/jpa

Commit:
[DD-898] upgraded hibernate-validator + hibernate-maven-plugin versions

Commit:
[DD-898] upgraded el-api + maven-pmd-plugin version

Commit:
[DD-898] added some more domain model POJO classes to persistence unit + added a hibernate property to JPA persistence unit properties

knutwalker committed:
[DD-898] Use custom JPA Merge Event Listener

knutwalker committed:
[DD-898] Upgrade Hibernate to latest 4.3.x

Commit:
Merge branch 'sprint-29/dd-898' of github.com:knutwalker/dswarm into sprint-29/dd-898

Conflicts:
	persistence/pom.xml

Commit:
[DD-898] corrected persistence module pom after merge

Commit:
[DD-898] added fix for depracated hibernate persistence provider

Commit:
[DD-898] removed persistence provider service from META-Inf in persistence module

Commit:
[DD-898] upgraded to JPA 2.1 + fixed some deprecated method usages

Commit:
[DD-898] upgraded various maven plugins versions

Commit:
[WIP][DD-551] upgrade to java 8

Commit:
[WIP][DD-551] fixed CSVTransformationFlowTest failures

Commit:
[WIP][DD-551] fixed MorphScriptBuilderTest failures

Commit:
[DD-910] added Marc21 schema generation (from XSD) initially

Commit:
[DD-910] added content schema to marc21 schema generation

Commit:
[DD-910] upgraded init metadata repository sql script

Commit:
[DD-910] fixed init script re. utilised uuids of tests

Commit:
[DD-910] enabled marc21 for separate storage_type processing

knutwalker committed:
Merge pull request #19 from zazi/sprint-29/dd-898

[DD-514] / [DD-898] upgrade hibernate + JPA versions

knutwalker committed:
Merge pull request #26 from zazi/sprint-29/dd-551

[DD-514] + [DD-551] upgrade hibernate + jpa + java

knutwalker committed:
Merge pull request #25 from zazi/sprint-29/dd-910

[DD-910] added Marc21 to init metadata repository sql script

Commit:
Revert "Revert "[DD-392] schema attribute path instance extension + re-factoring""

This reverts commit 3d4588250b9d787660fb7a4e68d16e8d909339ae.

Commit:
Merge branch 'builds/unstable' into new/builds/unstable

dswarm-jenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://sdvjenkins.slub-dresden.de/job/Github%20PR%20Datamanagement%20Platform/22/<h2>Failed Tests: <span class='status-failure'>4</span></h2><h3><a name='org.dswarm:dswarm-controller' /><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/22/org.dswarm$dswarm-controller/testReport'>org.dswarm:dswarm-controller</a>: <span class='status-failure'>4</span></h3><ul><li><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/22/org.dswarm$dswarm-controller/testReport/org.dswarm.controller.resources.resource.test/DataModelsResourceTest/testExportDataModelAsUnsupportedFormat/'><strong>org.dswarm.controller.resources.resource.test.DataModelsResourceTest.testExportDataModelAsUnsupportedFormat</strong></a></li><li><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/22/org.dswarm$dswarm-controller/testReport/org.dswarm.controller.resources.resource.test/DataModelsResourceTest/testMABXMLData/'><strong>org.dswarm.controller.resources.resource.test.DataModelsResourceTest.testMABXMLData</strong></a></li><li><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/22/org.dswarm$dswarm-controller/testReport/org.dswarm.controller.resources.resource.test/DataModelsResourceTest/testExportDataModelAsRandomFormat/'><strong>org.dswarm.controller.resources.resource.test.DataModelsResourceTest.testExportDataModelAsRandomFormat</strong></a></li><li><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/22/org.dswarm$dswarm-controller/testReport/org.dswarm.controller.resources.resource.test/DataModelsResourceTest/testPUTObject/'><strong>org.dswarm.controller.resources.resource.test.DataModelsResourceTest.testPUTObject</strong></a></li></ul>
Test FAILed.

Commit:
tried to switch to eclipselink persistence provider

Commit:
switch from HQL queries to JPQL queries

Commit:
tried to get rid of CloneNotSupportExceptions ...

Commit:
upgraded eclipselink version

knutwalker committed:
[DD-919] Update Metrics with advanced features

- Use metrics annotations and implement @Timed and @Metered for
  Jersey Resources in an awesome fashion (hint inherited annotation)

knutwalker committed:
Remove situation in where Schema#APs could be null

Conflicts:
	persistence/src/test/java/org/dswarm/persistence/model/schema/test/SchemaTest.java

knutwalker committed:
[DD-514] Use new Hibernate entity copy observer feature

Commit:
Merge pull request #29 from knutwalker/builds/unstable

[DD-514]

Commit:
fixed blob bug

Commit:
[DD-921] fixed schema sql script (re. blob) + disabled functions.sql execution at init-db (because they are already in schema.sql)

knutwalker committed:
[DD-919] Add JVM instrumentation

knutwalker committed:
[DD-919] Fix raced creation of MetricRegistries

Commit:
[DD-921] removed eclipselink specific caching settings

Commit:
[DD-921] utilise merged object, when updating an object in metadata repository

Commit:
Merge remote-tracking branch 'dswarm-phorn/sprint-30/dd-919' into eclipselinknew2

Commit:
[WIP][DD-921] towrds simplified create object method in controller servies (re-worked attribute + clasz creation in perssitence layer)

Commit:
[WIP][DD-921] replaced em.persist with em.merge for object creation

Commit:
[WIP][DD-921] fixed attribute/class entity creation

Commit:
[WIP][DD-921] introduced static weaving for eclipselink

Commit:
[WIP][DD-921] fixed test in converter module after static weaving migration

Commit:
[DD-921] provided custom deserializer for Resource and Configuration sets (that resolve entity references to full objects (if available))

Commit:
[DD-921] some code clean-ups re. various refactorings

dswarm-jenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://sdvjenkins.slub-dresden.de/job/Github%20PR%20Datamanagement%20Platform/31/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.dswarm:dswarm-controller' /><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/31/org.dswarm$dswarm-controller/testReport'>org.dswarm:dswarm-controller</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/31/org.dswarm$dswarm-controller/testReport/org.dswarm.controller.resources.job.test/ProjectsResourceTest/testGETObject/'><strong>org.dswarm.controller.resources.job.test.ProjectsResourceTest.testGETObject</strong></a></li></ul>
Test FAILed.
